viewer/packages/cad-parsers/src/cad/V9GeometryFilterer.ts

```diff
@@ -0,0 +1,163 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { assertNever, TypedArray, TypedArrayConstructor } from '@reveal/utilities';
+import { RevealGeometryCollectionType } from '@reveal/sector-parser';
+import { computeBoundingBoxFromEllipseValues } from '../utilities/computeBoundingBoxFromAttributes';
+import { filterPrimitivesOutsideClipBox } from './filterPrimitivesCommon';
+import * as THREE from 'three';
+
+export class V9GeometryFilterer {
```

Comment:
Not sure it makes sense to have this as a class since it only has static functions.